knnvector/runners.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import time
+
+from opensearchpy.exceptions import ConnectionTimeout
+
+from knnvector.tools.parse import parse_int_parameter, parse_string_parameter
+
+
+def register(registry):
+    registry.register_runner(
+        "custom-vector-bulk", BulkVectorsFromDataSetRunner(), async_runner=True
+    )
+    registry.register_runner(
+        "warmup-indices", WarmupIndicesRunner(), async_runner=True
+    )
+
+
+# TODO: Replace BulkVectorsFromDataSetRunner with BulkIndexer
```

Comment:
Could you add reasoning why we dont need this? I believe its because we should not need retry logic

Reply:
Sure.